cypress/support/utils.js

```diff
@@ -181,8 +193,10 @@ export const labelDelete = (groupName, labelName) => {
       cy.get('div[data-testid="column"]')
         .contains("🗑️")
         .should("exist")
-        .click();
+        .click({ force: true });
     });
+
+  cy.log("*** labelDelete (end): groupName", labelName);
```

Comment:
nit: do we want to keep this log?

Reply:
will remove after we are done debugging.